src/views/GameDetails/GameServer/GameServer.scss

```diff
@@ -0,0 +1,101 @@
+@import "../../../index";
+
+.template-title {
+    color: $color-white;
+    align-self: center;
+    font-size: 2rem
+}
+
+.server-container {
+    display: flex;
+    justify-content: space-evenly;
+    
+    div {
+        min-width: 400px;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .input-field {
+        display: flex;
+        flex-direction: row;
```

Comment:
The input field doesn't really need to be a flex box, so these lines can be removed as far as I can tell.

Reply:
Removed